### ISSUE TITLE
Android Studio preview not working

### ISSUE THREAD

**Issue by robertdolca** [uber/rides-android-sdk#13]
Error `Failed to find style 'uberButtonStyle' in current theme`

Code used:
```
                <com.uber.sdk.android.rides.RequestButton
                    android:id="@+id/uber_button"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    uber:style="white" />
```

**Comment by ritviksachdev** [comment]
Hey robert,

In the parent layout of the request button have you included the xml namespace for uber?
An example would be
`
<LinearLayout
   xmlns:android="http://schemas.android.com/apk/res/android"
   **xmlns:uber="http://schemas.android.com/apk/res-auto"**
   android:layout_width="match_parent"
   android:layout_height="match_parent">

<com.uber.sdk.android.rides.RequestButton
                    android:id="@+id/uber_button"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    uber:style="white" />

</LinearLayout>
`

**Comment by robertdolca** [comment]
I've added the namespace to the root element in the layout next to `xmlns:android`. Is that a problem?

**Comment by ritviksachdev** [comment]
As mentioned in the documentation:

**To use the `uber` custom attribute be sure to add `xmlns:uber="http://schemas.android.com/apk/res-auto"` to your root view element.**

Can you verify that you have added the namespace to your root view element?

**Comment by robertdolca** [comment]
@ritviksachdev Thanks for taking the time to investigate.

I created a simple layout:

```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
              xmlns:uber="http://schemas.android.com/apk/res-auto"
              android:orientation="vertical"
              android:layout_width="match_parent"
              android:layout_height="match_parent">
    <com.uber.sdk.android.rides.RequestButton
        android:id="@+id/uber_button"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        uber:style="white" />
</LinearLayout>
```

The error is still the same.

I do not believe that this should influence but I excluded a dependency from the Uber sdk because of a conflict.

```
    compile ('com.uber.sdk:rides-android:0.2.0') {
        exclude group: 'com.google.http-client', module: 'google-http-client-jackson2'
    }
```

**Comment by itstexter** [comment]
Hey robertdolca,

When you say Preview do you mean the Sample app in releases? It looks like your app is having trouble finding the internal uberButtonStyle used for the Button.

If you aren't using the sample app, could you try to run the sample app in your environment?  I'd be curious to see if the same error happens, if not, that might narrow things down a bit.

**Comment by robertdolca** [comment]
When I say preview I mean Android Studio layout preview. The app works perfectly fine when it is run on a device. I will try the sample app.

**Comment by itstexter** [comment]
@robertdolca Any success?

**Comment by robertdolca** [comment]
@itstexter I integrated the button using `RequestDeeplink.Builder` directly and a "custom design" and everything works fine.

I reproduced the preview error in Android Studio when using `<com.uber.sdk.android.rides.RequestButton` with the sample app.

**Comment by robertdolca** [comment]
I was curious what was the root cause. In order to make the preview work with a custom app theme you need to add this in your app's theme

```
<item name="uberButtonStyle">@style/UberButton</item>
```

I coudn't find this information in the documentation. I would like to add it there in the README.

**Comment by tyvsmith** [comment]
Thanks for reporting @robertdolca! We've identified the bug and will be fixing.

**Comment by arogal** [comment]
This is now fixed in 0.3.2. Thanks!
